.changeset/angry-pots-boil.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': patch
+'@astrojs/mdx': patch
+'@astrojs/markdown-remark': patch
+---
+
+Add SmartyPants back to Astro's default Markdown and MDX plugins. This can be disabled using the `markdown.smartypants` config option:
```

Comment:
I would reword this changelog from the perspective of migrating from Astro 1.0 => Astro 2.0!

Reply:
Done!